src/internal/kopia/wrapper.go

```diff
@@ -228,8 +210,69 @@ func (w Wrapper) ConsumeBackupCollections(
 	return s, progress.deets, progress.toMerge, progress.errs.Failure()
 }
 
+type userAndHost struct {
+	user string
+	host string
```

Comment:
`host` is an opaque concept from my perspective.  Could you please add a comment either here on the property or on the struct itself explaining its composition, identity, purpose, etc?  I think that'd help maintainers in the long run.